common/brave_switches.cc

```diff
@@ -40,4 +40,12 @@ const char kUiMode[] = "ui-mode";
 // available.
 const char kUpgradeFromMuon[] = "upgrade-from-muon";
 
+// Allows disabling the machine ID generation on Windows.
+const char kDisableMachineId[] = "disable-machine-id";
```

Comment:
can you do another chromium_src override to add these to base_switches.h so we don't duplicate the string literals? Right now these aren't actually used anywhere

Reply:
Yes, I was wondering how I was going to do that, I see clearly now :upside_down_face:

Reply:
@bridiver Hum I'm strungling creating a base_switches override in chromium_src and using it. There is a sort of cyclomatic inclusion and don't know how to solve it properly. Can you lead me on this ?

Reply:
@bridiver Maybe I could do something like @darkdh does in 0c1a3b6 ?

Reply:
sure, that works too

Reply:
but for the base_switches you would basically just be defining new switches in the same namespace and then re-including the cc and h files

Reply:
I have made both actually :

![image](https://user-images.githubusercontent.com/1951866/51066963-1b024e80-160e-11e9-8662-06c041251c1d.png)

![image](https://user-images.githubusercontent.com/1951866/51066971-22295c80-160e-11e9-9030-dd169897efa5.png)

![image](https://user-images.githubusercontent.com/1951866/51066994-4e44dd80-160e-11e9-997f-49ae481dc9fe.png)

Reply:
@bridiver I have push some changes to reduce potential conflicts. But cannot figure out for switches unfortunately if you want to made those changes for me :)